lib/puppet/file_bucket/file.rb

```diff
@@ -79,6 +78,10 @@ def self.from_s(contents)
     self.new(contents)
   end
 
+  def self.from_binary(contents)
+    self.new(contents)
+  end
+
```

Comment:
Removing `from_s` makes sense to me, but `to_s` is core Ruby functionality, and a ton of tests seem to rely on Ruby automagically turning the `Puppet::FileBucket::File` object into a string.